logd/store/cursor.go

```diff
@@ -0,0 +1,75 @@
+package store
+
+import (
+	"encoding/base32"
+	"errors"
+	"fmt"
+
+	"github.com/deref/exo/util/bitutil"
+)
+
+var encoding32 = base32.NewEncoding("0123456789abcdefghjkmnpqrstvwxyz").WithPadding(base32.NoPadding)
+
+type Cursor struct {
+	ID        []byte
+	Direction direction
```

Comment:
nitpicky, but I think the order of these two fields should be reversed. This way it mirrors the encoding & b/c the direction is dominate in the hierarchy of cursors.